libraries/GCS_MAVLink/GCS.h

```diff
@@ -458,9 +456,6 @@ class GCS_MAVLINK
     uint32_t        waypoint_timelast_request; // milliseconds
     const uint16_t  waypoint_receive_timeout = 8000; // milliseconds
 
-    // number of 50Hz ticks until we next send this stream
-    uint8_t         stream_ticks[NUM_STREAMS];
-
     // number of extra ticks to add to slow things down for the radio
     uint8_t         stream_slowdown;
```

Comment:
stream_slowdown is actually a number of milliseconds now isn't it?  so we could name it stream_slowdown_ms perhaps?  .. or maybe just fix the comments if it's not "ticks" anymore.

Reply:
It's units are actually 50ths of a second... fixed the comment.